From 4be71525f07189e97720688472172604879d49a5 Mon Sep 17 00:00:00 2001
From: rvalue <i@rvalue.moe>
Date: Fri, 17 Dec 2021 12:26:42 +0000
Subject: [PATCH] Remove incorrect args for i18n.merge_file

`i18n.merge_file` has been ignoring positional arguments for a time and
explicitly rejects with error since meson 0.60.0
---
 data/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index f6692b6..6981b96 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -59,7 +59,6 @@ endif
 appdata_conf = configuration_data()
 appdata_conf.set('app_id', weather_id)
 appdata = i18n.merge_file(
-  'appdata',
   input: configure_file(
     input: '@0@.appdata.xml.in.in'.format(default_id),
     output: '@0@.appdata.xml.in'.format(default_id),
-- 
GitLab

